src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const isActiveTabPresent = tabs.some(tab => tab.id === selectedTabId);
+  const verifiedSelectedTabId = isActiveTabPresent ? selectedTabId : tabs[0].id;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab, index) => (
```

Comment:
```suggestion
          {tabs.map((tab) => (
```
Why do you need index? You don't use it

Reply:
True, removed the index